### DISCUSSION TITLE
Associated dates that can be published in a calendar (ics availability of items)

### DISCUSSION THREAD

**Discussion by alexanderadam** [sysadminsmedia/homebox#739] [Ideas]
### What is the problem you are trying to solve with this feature?

It's easy to forget about scheduled maintenance, expiring warranty, device checks or even historic events (i.e. for insurance) etc

### What is the solution you are proposing?

A "dates" area where a list of timestamps with respective labels can be maintained by item.
For instance

```yaml
warranty expiration: 12.05.2028
handle broke in the storm: 03.05.2022
```

These dates could be exposed as ICS at a certain path so that people can just integrate them in their calendar.
The scheduled maintenance dates should be in that ICS as well.

### What alternatives have you considered?

🤷 

### Contributions

- [x] I have searched through existing issues and feature requests to see if my idea has already been proposed.
- [ ] If this feature is accepted, I would be willing to help implement and maintain this feature.
- [ ] If this feature is accepted, I'm willing to sponsor the development of this feature.

**Comment by tankerkiller125** [comment]
Please note we're converting this issue into a discussion as per our new procedures/policies. You can read more about that in our announcement #712
